kwave/utils/atten_comp.py

```diff
@@ -67,6 +67,8 @@ def atten_comp(
     # extract signal characteristics
     N, num_signals = signal.shape
 
+    assert y != 1, "A power exponent [y] of 1 is not valid." 
```

Comment:
If this is a user input, maybe use `if-else / raise` statements?

Reply:
Assuming `if-else / raise` statements will also remove the warning.

Reply:
done.